datasource_secrets.go

```diff
@@ -4,10 +4,10 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"fmt"
+	"github.com/sspinc/terraform-provider-credstash/credstash"
 	"log"
 
 	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/sspinc/terraform-provider-credstash/credstash"
```

Comment:
Please revert re-ordering the imports. Grouping of the imports is important. I think one of the automatic formatters does it out of the box.